docs/usage.md

```diff
@@ -0,0 +1,61 @@
+# Importing Type Definitions
+
+This guide assumes you've followed the installation steps in the [Quick start guide](quickstart.md), and that you've installed the the definitions for your project's dependencies.
+
+## Existing Definitions
+
+> Using the `/flow-typed` directory for `libdefs` is a convention that enables Flow to JustWork™ out of the box and encourages consistency across projects that use Flow, but it is also possible to explicitly configure Flow to look elsewhere for `libdefs` using the `[libs]` section of your `.flowconfig`.[ [1] ](https://flow.org/en/docs/libdefs/)_
```

Comment:
Is this extra underscore a mistake from ages ago?
```suggestion
> Using the `/flow-typed` directory for `libdefs` is a convention that enables Flow to JustWork™ out of the box and encourages consistency across projects that use Flow, but it is also possible to explicitly configure Flow to look elsewhere for `libdefs` using the `[libs]` section of your `.flowconfig`.[ [1] ](https://flow.org/en/docs/libdefs/)
```

Reply:
Looks like it was 👀